src/features/applications/types/index.ts

```diff
@@ -29,14 +34,28 @@ export const fundingSourceTypes = {
   OTHER: "Other",
 } as const;
 
+export const fundingAmountTypes = {
+  HUGE: "Greater than $1M USD",
+  ALOT: "1M - 500K USD",
+  LOT: "500K - 250K USD",
+  MLOT: "250K - 100K USD",
+  LITTLE: "100K - 10K USD",
+  SMALL: "10K-1K USD",
+  TINY: "Less than 1K USD",
+} as const;
+
 export const ApplicationSchema = z.object({
-  name: z.string().min(3),
+  // name: z.string().min(3),
   bio: z.string().min(3),
   websiteUrl: z.string().url().min(1),
-  payoutAddress: EthAddressSchema,
+  payoutAddress: z.string().optional(),
+  githubProjectLink: z.string().url().includes("https://github.com/").min(20),
+
   contributionDescription: z.string().min(3),
   impactDescription: z.string().min(3),
   impactCategory: z.array(z.string()).min(1),
+  teamDescription: z.string().min(3),
+  twitterPost: z.string().url().includes("https://x.com/").min(15).optional(),
```

Comment:
what if the user posts pointing to twitter.com? Might be good to add an error message: 
```ts
z.string({ message: "Must be in the format of https://x.com/... })
```

Reply:
includes brings the error `Invalid input: must include `https://x.com/`